bundles/org.openhab.binding.bigassfan/src/main/java/org/openhab/binding/bigassfan/internal/handler/BigAssFanHandler.java

```diff
@@ -324,8 +330,10 @@ private void adjustMaxSpeed(PercentType command, String channelId, String comman
     private void adjustMinSpeed(PercentType command, String channelId, String commandFragment) {
         int newMax = command.intValue();
         int currentMin = PercentType.HUNDRED.intValue();
-        if (fanStateMap.get(channelId) != null) {
-            currentMin = ((PercentType) fanStateMap.get(channelId)).intValue();
+        @Nullable
```

Comment:
This is not needed here.

Reply:
Not yet resolved, so reopening.

Reply:
Not intentionally, sorry. just fixed it.